EDDiscovery/EDDiscovery.csproj

```diff
@@ -243,7 +243,6 @@
       <SubType>Component</SubType>
     </Compile>
     <Compile Include="EDDProfiles.cs" />
-    <Compile Include="Properties\ExtraVersion.cs" />
```

Comment:
People really don't want build versioning, do they?

Reply:
Just add in an empty file with the right name locally - it's included in the .gitignore so you won't commit the file itself but you'll stop the .csproj change being included in the commit.